### DISCUSSION TITLE
Vscode version 1.6 does not work with github copilot

### DISCUSSION THREAD

**Discussion by nervusdm** [coder/code-server#5213] [Q&A] [answered]
Hello,

Since vscode "server" is 1.66.2 :

**Unable to install extension 'github.copilot' as it is not compatible with VS Code '1.66.2'.**

Do you guys know when a update will be available or a quick fix ?

Thanks :)

**Comment by jsjoeio** [comment]
We're a little behind on the Code updates but keep an eye on the May milestone!

**Comment by craftxbox** [comment, marked as answer]
If you unpack the latest VSIX files from the VSC marketplace, You can fudge the package manifests into allowing the extension to run on 1.66.2. 
On inspection there are three places that the version is specified:
Once in ./extension/dist/extension.js
Once in ./extension/package.json
Once in ./extension.vsixmanifest
Change all those occurances, then repackage it into a zip with the .vsix extension, And it should install and work without any issues.
On linux & the current latest build of copilot targetting 1.67.0, this looks somewhat like:
_(untested, for reference only. execute at your own risk)_
```sh
unzip copilot-original.vsix -d copilot.tmp
cd copilot.tmp
find . -type f -exec sed -i "s|\\^1\\.67\\.0|^1.66.0|g" {} \;
zip -r ../copilot.vsix *
cd ..
rm -rf copilot.tmp
```
Then just install the resulting VSIX and authenticate as normal.

**Reply by nervusdm** [reply]
I removed -type=f and it's fine thx

**Reply by 10t4p3b3** [reply]
i have version 1.68 and it says: user not authorized. what do you suggest me to do?

**Reply by craftxbox** [reply]
Copilot appears to be going through a kindof re-structuring right now, Probably best to give it some time.

**Reply by nervusdm** [reply]
How can you use version 1.68 ? When I install copilot, I haven't this version.

Copilot is working fine with this "paid version", just do the same thing changing the number version

**Reply by craftxbox** [reply]
Youll need to change the version manually based on what's actually there, The script is only provided for reference as the version numbers change all the time.
